client/js/components/user/DpOrganisationList/DpOrganisationList.vue

```diff
@@ -312,13 +312,13 @@ export default {
   },
 
   computed: {
-    ...mapState('orga', {
+    ...mapState('Orga', {
```

Comment:
When I open this page, there are a lot of errors in the console and the list of organisations is not displayed

Reply:
should be fixed with 852ab24aea8726fda86706a749b7c6bd90ed2269